### PR TITLE
feat!: migrate to GitHub

### PR DESCRIPTION
**Pull request by sanakhandvsa**
## Description

Related issue: [JIRA_TICKET_NUMBER](LINK_TO_JIRA_TICKET)

## Before submitting (or marking as "ready for review")

- [ ] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [ ] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,59 +1,62 @@
...
     "autoload": {
         "psr-4": {
             "Dvsa\\Olcs\\Auth\\": "src/"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+          "OlcsTest\\Logging\\": "test/"
+        }
+    },
```
Comment (JoshuaLicense) [inline composer.json:33]: This is for the `olcs-logging` namespaces.

File: test/src/ControllerFactory/LogoutControllerFactoryTest.php [hunk 2]
```diff
@@ -1,41 +1,53 @@
 <?php
 
-namespace Dvsa\OlcsTest\Auth\ControllerFactory;
+namespace OlcsTest\Logging\src\ControllerFactory;
```
Comment (JoshuaLicense) [inline test/src/ControllerFactory/LogoutControllerFactoryTest.php:3]: This is a `olcs-logging` namespace.

File: src/ControllerFactory/ExpiredPasswordControllerFactory.php [hunk 3]
```diff
@@ -27,8 +27,6 @@ class ExpiredPasswordControllerFactory implements FactoryInterface
      * @param $requestedName
      * @param array|null $options
      * @return ExpiredPasswordController
-     * @throws ContainerExceptionInterface
-     * @throws NotFoundExceptionInterfaceAlias
```
Comment (JoshuaLicense) [inline src/ControllerFactory/ExpiredPasswordControllerFactory.php:31]: Does this method no longer throw these exceptions?

File: composer.json [hunk 4]
```diff
@@ -1,59 +1,67 @@
...
+      "php": "^7.4",
+        "laminas/laminas-mvc": "^2.7.14 || ^3.0",
+        "laminas/laminas-session": "^2.8.1",
+        "laminas/laminas-stdlib": "~2.7.9",
+        "laminas/laminas-http": "^2.5",
+        "laminas/laminas-servicemanager": "^2.7.5 || ^3.0.3",
+        "laminas/laminas-view": "^2.11",
+        "laminas/laminas-authentication": "^2.5",
+        "laminas/laminas-form": "^2.7",
+        "container-interop/container-interop": "^1.2"
```
Comment (JoshuaLicense) [inline composer.json:16]: Could we run `--sort-packages` for Composer so that these are sorted correctly please?

**Review by JoshuaLicense — CHANGES_REQUESTED** [review]
`prefer-lowest` and `prefer-stable` are installing the same Laminas MVC version (2.7.X) - be good to find out what's blocking the install to Laminas MVC `^3.0`.

**Review by fibble — APPROVED** [review]

**Pull request merged**